app/styles/partials/overrides.scss

```diff
@@ -35,16 +35,38 @@ body.dimmable.undetached.dimmed {
   }
 }
 
-.margin.less {
-  padding: 0 !important;
+.less {
+  &.margin {
+    margin: 0 !important;
+  }
+
+  &.bottom.margin {
+    margin-bottom: 0 !important;
+  }
+
+  &.top.margin {
+    margin-top: 0 !important;
+  }
 }
 
 .less {
   &.padding {
     padding: 0 !important;
   }
-  
-  &.padding-right {
+
+  &.right.padding {
```

Comment:
@CosmicCoder96 @niranjan94 I am not able to understand how using this convention will solve the problem which you've mentioned in the above comment? Can you please clarify and make it clear?